ui/layout.js

```diff
@@ -91,6 +94,20 @@ class OverviewClone extends St.BoxLayout {
             Main.overview.show();
         });
         this.add_action(clickAction);
+
+        this._extensionStateChangedId =
+        Main.extensionManager.connect('extension-state-changed',
+            () => OverviewOverrides.updateGhostPanelPosition(box));
+        OverviewOverrides.updateGhostPanelPosition(box);
```

Comment:
Maybe on extension.js we should change the order where `Layout.enable()` and `Overview.enable()` are invoked as layout is depending on overview... although for this specific change it shouldn't matter as `Overview.updateGhostPanelPosition()` doesn't depend on `Overview.enable()` being invoked.